app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -114,4 +115,12 @@ public void onDestroyView() {
             disposable.dispose();
         }
     }
+
+    void onPlayPause() {
+        if(controller == null) {
+            return;
+        }
+        controller.init();
```

Comment:
Init should only be called one single time when constructing the controller. That is done in OnStart. Please add a space between if and (

Reply:
Let me make the the necessary adjustment.